sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1292,6 +1293,11 @@ class RecognizeEntitiesAction(DictMixin):
         `UnicodeCodePoint`, the Python encoding, is the default. To override the Python default,
         you can also pass in `Utf16CodePoint` or TextElements_v8`. For additional information
         see https://aka.ms/text-analytics-offsets
+    :ivar str model_version: The model version to use for the analysis.
```

Comment:
should we link here for model version?

https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning

Reply:
Why not, thanks!